projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-navigator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Launchpad API endpoint
+ *
+ * @package automattic/jetpack-mu-wpcom
+ * @since 1.1.0
+ */
+
+/**
+ * Fetches Launchpad Navigator-related data for the site.
+ *
+ * @since 1.1.0
```

Comment:
As above RE the version.
```suggestion
 * @since 4.9.0
```